schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -2461,6 +370,2119 @@
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/routeTables_routes" },
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/securityPartnerProviders" },
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/serviceEndpointPolicies" },
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/serviceEndpointPolicies_serviceEndpointPolicyDefinitions" },
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/virtualHubs" },
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/virtualHubs_bgpConnections" },
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/virtualHubs_hubRouteTables" },
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/virtualHubs_hubVirtualNetworkConnections" },
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/virtualHubs_ipConfigurations" },
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/virtualHubs_routeTables" },
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/virtualNetworkGateways" },
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/virtualNetworks" },
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/virtualNetworks_subnets" },
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/virtualNetworks_virtualNetworkPeerings" },
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/virtualNetworkTaps" },
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/virtualRouters" },
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/virtualRouters_peerings" },
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/virtualWans" },
+                { "$ref": "https://schema.management.azure.com/schemas/2020-05-01/Microsoft.Network.json#/resourceDefinitions/vpnGateways" },
+                { "$ref
```

Comment:
This should be `items` rather than `additionalProperties` to declare the schema for array contents.

Reply:
Good catch. Fixed.